packages/nextjs/pages/api/etl/index.ts

```diff
@@ -267,21 +271,29 @@ const getMovement = (current: number, comparison: number) => {
   return diff;
 };
 
-const getImpactIndex = (project: Metrics, weights: Metrics): number => {
+const getImpactIndex = (
+  project: Metrics,
+  weights: Metrics,
+  maxScoreByMetric: { [key in keyof Metrics]: number },
+): number => {
   let impact_index = 0;
-  const metricKeys = Object.keys(weights) as (keyof Metrics)[];
+  const metrics = Object.keys(maxScoreByMetric) as (keyof Metrics)[];
   let weightSum = 0;
-  for (const metric of metricKeys) {
+  for (const metric of metrics) {
     weightSum += weights[metric];
   }
   // If the weights don't add up to 100 then we expect they are each a 0-100 percentage and need to be adjusted
   const indPct = Math.round(weightSum) !== 100;
-  for (const metric of metricKeys) {
+  for (const metric of metrics) {
     const multiplier = indPct ? weights[metric] / weightSum : weights[metric];
```

Comment:
Remove the condition logic here. This is left over from before we knew the format of the weights.
You can remove line 286 and make this line look like this:
```javascript
const multiplier = weights[metric] / weightSum;
```

Reply:
Makes sense, thanks